internal/cli/args_test.go

```diff
@@ -0,0 +1,78 @@
+package cli
+
+import (
+	"os"
+	"testing"
+)
+
+func TestHandleArgs(t *testing.T) {
+	tests := []struct {
+		name     string
+		args     []string
+		version  string
+		expected bool
+	}{
+		{
+			name:     "no flags",
+			args:     []string{"neo4j-mcp"},
+			version:  "1.0.0",
+			expected: false,
+		},
+		{
+			name:     "version flag short form",
+			args:     []string{"neo4j-mcp", "-v"},
+			version:  "1.0.0",
+			expected: true,
+		},
+		{
+			name:     "version flag long form",
+			args:     []string{"neo4j-mcp", "--version"},
+			version:  "1.0.0",
+			expected: true,
+		},
+		{
+			name:     "help flag short form",
+			args:     []string{"neo4j-mcp", "-h"},
+			version:  "1.0.0",
+			expected: true,
+		},
+		{
+			name:     "help flag long form",
+			args:     []string{"neo4j-mcp", "--help"},
```

Comment:
we probably want to test for the help text here instead of the version?
you can check coverage via `go test -coverprofile cover.out` or right click somewhere inside the file in vscode and click on `toggle unit test coverage for this file` or something like that